boards/x86/acrn/doc/index.rst

```diff
@@ -0,0 +1,30 @@
+.. _acrn:
+
+ACRN UOS (Guest Operating System)
+#################################
+
+Overview
+********
+
+This baseline configuration is intended to be used as a starting point for
```

Comment:
This could use more introduction since project ACRN is likely an unknown thing to most Zephyr developers.

Something like this perhaps:
```
This board configuration defines an ACRN User OS execution environment for
running Zephyr RTOS applications.

ACRN is a flexible, lightweight reference hypervisor, built with real-time
and safety-criticality in mind, optimized to streamline embedded
development through an open source platform. Check out the
`Introduction to Project ACRN <https://projectacrn.github.io/latest/introduction/>`_
for more information.

This baseline configuration can be used as a starting point for creating
custom ACRN UOS configurations.  It currently supports the following devices: 
```